### PR TITLE
Add raft

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

File: examples/raft/tests/custom_test_tensorleap.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    x = subset_images()
+    # get inputs
+    img_1 = np.expand_dims(input_image1(i, scene_flow), axis=0)
+    img_2 = np.expand_dims(input_image2(i, scene_flow), axis=0)
+    # run inference on Onnx
+    if not exists("raft.onnx"):
+        print("Downloading Raft ONNX for inference")
+        urllib.request.urlretrieve(
+            "https://storage.googleapis.com/example-datasets-47ml982d/raft/raft_new.onnx",
+            "raft.onnx")
```
Comment (doronHarnoy) [inline examples/raft/tests/custom_test_tensorleap.py:24]: Is this a public bucket that anyone can reach?

File: examples/raft/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+**RAFT** is an optical flow model that uses an all-pairs recurrent model to approximate the optical flow solution.
+In this example, we infer [RAFT](https://github.com/princeton-vl/RAFT) on the
+[KITTI](https://www.cvlibs.net/datasets/kitti/index.php) dataset .
+
+**Prerequisites**
+
+Before you begin, ensure that you have the following prerequisites installed:
+
+- **[Python](https://www.python.org/)** (version 3.7 or higher)
```
Comment (doronHarnoy) [inline examples/raft/README.md:11]: I think we work with 3.8

**Review by doronHarnoy — COMMENTED** [review]
Cool

**Review by doronHarnoy — APPROVED** [review]

File: examples/raft/tests/custom_test_tensorleap.py (hunk 1 above)
Comment (tomkoren21) [reply examples/raft/tests/custom_test_tensorleap.py:24]: Yes

**Pull request merged**